platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SamlProtocol.java

```diff
@@ -489,9 +492,18 @@ public static EntityDescriptor createSpMetadata(
 
     entityDescriptor.getRoleDescriptors().add(spSsoDescriptor);
 
+    entityDescriptor.setCacheDuration(getCacheDuration());
+
     return entityDescriptor;
   }
 
+  static long getCacheDuration() {
```

Comment:
❓ Any reasons we don't simply compute this as a static constant and re-use instead of retrieving it each time?